src/controllers/dynakube/activegate/consts/consts.go

```diff
@@ -59,4 +59,7 @@ const (
 	GatewaySslMountPoint     = "/var/lib/dynatrace/gateway/ssl"
 	GatewayLogMountPoint     = "/var/log/dynatrace/gateway"
 	GatewayTmpMountPoint     = "/var/tmp/dynatrace/gateway"
+
+	DockerImageUser  int64 = 1001
+	DockerImageGroup int64 = 1001
```

Comment:
just curios, why is this 1001 while the agent is 1000 ?

Reply:
no idea